.pre-commit-config.yaml

```diff
@@ -1,5 +1,21 @@
-exclude: ^(?:.+/)?\.idea(?P<ps_d>/).*$
```

Comment:
Was this done on purpose? Changes to this file seem to be purely organizational.  Are there any functional changes?